[Loggable] Allow composite identifiers (#2171)
doctrine-extensions/DoctrineExtensions: CHANGELOG.md

---

## [Unreleased]
### Added

> - Loggable: Support for composite identifiers

Reviewer: ```suggestion
- Loggable: Support for composite identifiers.
```

### Added
- Support for doctrine/dbal 3.x
- Timestampable: Support to use annotations as attributes on PHP >= 8.0.
- Loggable: Support for composite identifiers

### Deprecated

> - Implementing the `Gedmo\Tool\WrapperInterface::getIdentifier()` method second argument (`$flatten`) parameter is deprecated and will

Reviewer: ```suggestion
- Implementing the `Gedmo\Tool\WrapperInterface::getIdentifier()` method without the second argument (`$flatten`) is deprecated and will
```

- Support for doctrine/dbal 3.x
- Timestampable: Support to use annotations as attributes on PHP >= 8.0.

### Deprecated

> - Implementing the `Gedmo\Tool\WrapperInterface::getIdentifier()` method without the `$flatten` parameter is deprecated and will

Reviewer: ```suggestion
- Implementing the `Gedmo\Tool\WrapperInterface::getIdentifier()` method without the second argument (`$flatten`) is deprecated and will
```
